Source/Scene/Cesium3DTileStyle.js

```diff
@@ -246,13 +246,24 @@ define([
                 return this._show;
             },
             set : function(value) {
+                if (typeof value === 'boolean') {
+                    this._show = new Expression(String(value));
+                } else if (typeof value === 'string') {
+                    this._show = new Expression(value);
+                } else if (defined(value.conditions)) {
+                    this._show = new ConditionsExpression(value);
+                } else if (defined(value.expression) || defined(value.conditionsExpression)) {
+                    this._show = value;
+                } else {
+                    this._show = undefined;
+                    return;
+                }
                 this._showShaderFunctionReady = false;
-                this._show = value;
             }
         },
```

Comment:
For shader styling `undefined` is returned because otherwise `DEFAULT_JSON_COLOR_EXPRESSION` would cause all points to be white.

To fix this

```
that._colorShaderFunctionReady = !defined(styleJson.color);
that._showShaderFunctionReady = !defined(styleJson.show);
that._pointSizeShaderFunctionReady = !defined(styleJson.pointSize);
```

need to go below the setters in `setup`.